sale_blanket_order/models/blanket_orders.py

```diff
@@ -36,6 +36,9 @@ def _default_company(self):
     lines_ids = fields.One2many(
         'sale.blanket.order.line', 'order_id', string='Order lines',
         copy=True)
+    planned_order_ids = fields.One2many(
+        'sale.blanket.planned.order', 'blanket_order_id', string='planned Orders',
+        copy=True)
```

Comment:
Would put copy=False. If we duplicate an expired blanket order, no need to copy the planification

Reply:
Done!